SCons/CacheDirTests.py

```diff
@@ -34,6 +34,13 @@
 
 built_it = None
 
+IS_WINDOWS = sys.platform == 'win32'
```

Comment:
IS_WINDOWS is already in TestCmd..

% grep -r IS_WINDOWS testing/framework/*
testing/framework/TestCmd.py:IS_WINDOWS = sys.platform == 'win32'
testing/framework/TestSCons_time.py:from TestCmd import IS_WINDOWS
testing/framework/TestSCons_time.py:            if not IS_WINDOWS:

Maybe add IS_ROOT there as well?

Reply:
Somehow I thought only e2e tests used those definitions. But now I see it is used there, so that's a good change. Could add IS_ROOT, though it means it would be checked every time. Guess that's not a problem.

Reply:
I think that's fine. (checking for root every time..) It's not slow is it?

Reply:
> I think that's fine. (checking for root every time..) It's not slow is it?

It's an exception to handle every time the test framework starts up on Windows (since `os.geteuid` doesn't exist).  Lost in the noise, I'd say, but wanted to make sure the question was asked. Windows is our slow test platform, after all.

Reply:
`IS_ROOT` moved to framework after the chat on Discord.